src/yunohost/tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import sys
+import moulinette
+
+sys.path.append("..")
+
+old_init = moulinette.core.Moulinette18n.__init__
+
```

Comment:
I mean, it probably conflicts with it since it's merged, so probably need to remove it to solve the conflicts, but okay ;)